pkg/odo/util/completion/completion.go

```diff
@@ -159,3 +159,18 @@ func getUserTypedCommands(args complete.Args, command *cobra.Command) []string {
 
 	return commands
 }
+
+func AddProjectFlag(cmd *cobra.Command) {
```

Comment:
Can you add some comments about this and the following functions as well?
Them being in the completion package makes it a little tricky to immediately understand that they actually add a cobra flag without looking at the source code :)

Reply:
I agree. However the solution seems better than the alternative :)